File: TreasurersApp/ClientApp/src/app/models/EmailType.ts
export class EmailType {
  id: number = null;
  name: string = null;
  description: string = null;
  active: boolean = null;
  createdBy: string = null;
  createdDate: Date = null;
  lastModifiedBy: string = null;
  lastModifiedDate: Date = null;

  constructor(id: number,
    name: string,
    description: string,
    active: boolean,
    createdBy: string,
    createdDate: Date,
    lastModifiedBy: string,
    lastModifiedDate: Date) {
    this.id = id;
    this.name = name;
    this.description = description;
    this.active = active;
    this.createdBy = createdBy;
    this.createdDate = createdDate;
    this.lastModifiedBy = lastModifiedBy;
    this.lastModifiedDate = lastModifiedDate;
  }
}

File: TreasurersApp/Models/EmailType.cs
namespace TreasurersApp.Models
{
    using Newtonsoft.Json;

    public class EmailType
    {
        [JsonProperty("id")]
        public int EmailTypeId { get; set; } // EmailTypeID (Primary key)

        [JsonProperty("name")]
        public string Name { get; set; } // Name (length: 10)

        [JsonProperty("description")]
        public string Description { get; set; } // Description (length: 50)

        [JsonProperty("active")]
        public bool Active { get; set; } // Active

        [JsonProperty("createdBy")]
        public System.Guid CreatedBy { get; set; } // CreatedBy

        [JsonProperty("createdDate")]
        public System.DateTime CreatedDate { get; set; } // CreatedDate

        [JsonProperty("lastModifiedBy")]
        public System.Guid LastModifiedBy { get; set; } // LastModifiedBy

        [JsonProperty("lastModifiedDate")]
        public System.DateTime LastModifiedDate { get; set; } // LastModifiedDate
    }

}
